packages/base-asset/src/main.sw

```diff
@@ -66,7 +66,7 @@ impl SRC20 for Contract {
     /// }
     /// ```
     #[storage(read)]
-    fn total_supply(asset: AssetId) -> Option<u64> {
+    fn total_supply(_asset: AssetId) -> Option<u64> {
```

Comment:
This change should be removed. Any changes to the base asset contract need to be done in a separate PR

Reply:
Solved